### PR TITLE
Create an level index on cm_log.

### PR DESCRIPTION
**Pull request by fvovan**
https://github.com/cargomedia/cm/issues/2309

### CODE REVIEW

**Comment by fvovan** [comment]
I read documentation again, and  still don't completely understand how it actually works.
My assumption is that either for dev or production query
`db.cm_log.find({level  : { $in : [300] } }).sort({"_id" : -1}).skip(0).limit(50) ;`
fetches every document in the collection sorted by "_id", but then filter 50 docs  matching `{level  : { $in : [300] } }` criteria. 
it can explain why `{level  : { $in : [30011] } }` takes a while, but still no any visible reasons why `{level  : { $in : [300] } }` "real" query works in a shell but fails when executed with php.

The only detail that is different  is 'batchResult' setting for `MongoCursor`, but I can't imagine how it could affect the performance. May be query optimiser denies the plan I described  ?

Basically i'm still not sure that `{level: 1}` index will resolve a problem, but it definitely needs to be tried.

If it does not help, I have a suggestion to change sorting criteria to `{"createdAt" : -1}` so both query and sorting will be "covered" with single index and this way it could be more consistent in terms of executing with PHP driver.

@njam please review
cc @kris-lab

**Comment by njam** [comment]
lgtm
